Kontent.Ai.Management/Models/PreviewConfiguration/SpaceDomainRequestModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Kontent.Ai.Management.Models.Shared;
+using Newtonsoft.Json;
+
+namespace Kontent.Ai.Management.Models.PreviewConfiguration;
+
+/// <summary>
+/// Represents space domain request model.
+/// </summary>
+public class SpaceDomainRequestModel
```

Comment:
SDK does not use Request suffix. Also this model is identical to `SpaceDomainResponseModel` can we unify it?

Reply:
removed and unified